gdeploy/gdeploy

```diff
@@ -83,7 +83,7 @@ def check_ansible_installation():
     dist_type = dist()
     op_system = dist_type[0]
     if op_system != 'redhat':
-        return
+        return "Error: gdeploy is supported on RHEL6 or above"
```

Comment:
@chawlanikhil24 I need some change here. The calling function does not handle the return statement.
The idea is, on non-redhat systems we do not further checks to verify ansible installations. So we just return 
from the function.
You can just print an info statement. Like:

print "INFO: Non Red Hat system, no further ansible checks will be done"